README.md

```diff
@@ -578,20 +579,34 @@ See [#symbols](https://iexcloud.io/docs/api/#symbols) for detailed documentation
 
 ### Get Symbols for an Exchange
 
-Returns an array of symbols for a given exchange.
+Returns an array of symbols for a given exchange (requires the Market Identifier Code of an exchange).
```

Comment:
New API change requires the Market Identifier Code rather than the exchange's symbol. I guess IEX Cloud decided to add a breaking change without versioning or updating docs 😓.

Reply:
I think this should be "Returns an array of symbols for an exchange identified by a market identifier code."